commandapi-core/src/main/java/dev/jorel/commandapi/CommandAPIHandler.java

```diff
@@ -742,6 +748,23 @@ public void writeDispatcherToFile() {
 		}
 	}
 
+	<CommandSource> LiteralCommandNode<CommandSource> namespaceNode(LiteralCommandNode<CommandSource> original, String namespace) {
```

Comment:
Dealing with the rest of this review later, I just didn't want to use <Source> here because that would overwrite the class' parameter.
It is necessary to define an explicit parameter here because I think otherwise that resulted in an error. I did write this a few days ago so I am not sure about that but it definitely was a problem. I personally wouldn't change the parameter name here.

Making it static does make sense but I don't think calling `CommandAPIHandler.getInstance()` is a problem.

Reply:
Ah yeah, that's fair. 

https://github.com/JorelAli/CommandAPI/blob/d9ec96fe9c71271aae448cdaf2ba2c9a04829ebe/commandapi-core/src/main/java/dev/jorel/commandapi/CommandAPIHandler.java#L100-L116